substrate/frame/multisig/src/lib.rs

```diff
@@ -665,6 +691,25 @@ impl<T: Config> Pallet<T> {
 		signatories.insert(index, who);
 		Ok(signatories)
 	}
+
+	pub(crate) fn ensure_updated(id: &T::AccountId, call_hash: &[u8; 32], threshold: u16) -> bool {
```

Comment:
I think we can safely expose this function publicly as extrinsic to manually nudge it to speed up the process.